sidecar/config/config.go

```diff
@@ -358,7 +366,10 @@ func (c *Config) Validate() error {
 	}
 
 	if c.Proxy {
-		validators = append(validators, IsInSet("Rules service backend", c.RulesBackend, []string{Amalgam8Backend, KubernetesBackend}))
+		validators = append(
+			validators,
+			IsInSet("Rules service backend", c.RulesBackend, []string{Amalgam8Backend, KubernetesBackend}),
```

Comment:
Sure. That naming scheme was added by https://github.com/amalgam8/amalgam8/pull/430.